x/storage/keeper/keeper.go

```diff
@@ -1254,24 +1261,32 @@ func (k Keeper) CreateGroup(
 	store.Set(groupKey, k.groupSeq.EncodeSequence(groupInfo.Id))
 	store.Set(types.GetGroupByIDKey(groupInfo.Id), gbz)
 
+	addedMembersDetailEvent := make([]*types.EventGroupMemberDetail, 0, len(opts.Members))
 	// need to limit the size of Msg.Members to avoid taking too long to execute the msg
 	for _, member := range opts.Members {
 		memberAddress, err := sdk.AccAddressFromHexUnsafe(member)
 		if err != nil {
 			return sdkmath.ZeroUint(), err
 		}
-		err = k.permKeeper.AddGroupMember(ctx, groupInfo.Id, memberAddress)
+		err = k.permKeeper.AddGroupMember(ctx, groupInfo.Id, memberAddress, nil)
```

Comment:
A bit strange that we support create a group with initial `members` but without expire time, I suggest that we either dont support any member when create group or we should support expire time of member.

I prefer the first option

Reply:
agreed, actually cross-chain package is not allow create group with members, so we can make them has the same behavior